osu.Game/Skinning/LegacyDefaultComboCounter.cs

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Threading;
+using osuTK;
+
+namespace osu.Game.Skinning
+{
+    /// <summary>
+    /// Uses the 'x' symbol and has a pop-out effect while rolling over.
+    /// </summary>
+    public partial class LegacyDefaultComboCounter : LegacyComboCounter
```

Comment:
I'm honestly not sure this abstraction was required. Could have just copy-pasted and spent less effort? And potentially had more maintainable / legible code.

Reply:
Sure.

Reply:
No need to change it at this point, I'll do so if the review is painful enough.